model/MediaService.php

```diff
@@ -192,6 +208,17 @@ public function deleteResource(RdfResource $resource)
         return false;
     }
 
+    private function removeFromFilesystem($link): bool
+    {
+        $directory = dirname($link);
+
+        if ($directory !== '.') {
+            return $this->getFileManager()->deleteDirectory($directory);
+        } else {
+            return $this->getFileManager()->deleteFile($link);
+        }
```

Comment:
```suggestion
        if ($directory !== '.') {
            return $this->getFileManager()->deleteDirectory($directory);
        }
        
        return $this->getFileManager()->deleteFile($link);
```

You can avoid else always. It is never required. There are tons of benefits, but is mainly reduces code complexity, increases readability and helps with methods extraction :) 

https://beberlei.de/2020/02/25/clean_code_object_calisthenics_rules_i_try_to_follow.html
https://williamdurand.fr/2013/06/03/object-calisthenics/#2-dont-use-the-else-keyword